packages/bbui/src/Actions/autoresize_textarea.ts

```diff
@@ -1,10 +1,11 @@
-function resize({ target }) {
+function resize(e: Event) {
+  const target = e.target as HTMLTextAreaElement
   target.style.height = "1px"
   target.style.height = +target.scrollHeight + "px"
 }
 
-export default function text_area_resize(el) {
-  resize({ target: el })
+export default function text_area_resize(el: HTMLTextAreaElement) {
+  resize({ target: el } as any)
```

Comment:
does it have to be any?, if so can we leave a comment explaining why so a future typescripter wont be trying to type it?